.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,16 @@
+# Copyright The Veraison Go-COSE Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Same MPL issue: https://github.com/veraison/go-cose/blob/main/LICENSE

Reply:
Updated the license text.